config/routes.rb

```diff
@@ -3,4 +3,7 @@
   get 'users/:id', to: 'users#show', as: 'user'
   get 'users/:user_id/posts', to: 'posts#index', as: 'user_posts'
   get 'users/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
+
```

Comment:
- [OPTIONAL] Great work implementing the optional suggestion from the previous reviewer. Kindly remove these routes created because Rails resources will handle that. This will make your code less verbose and you would be adopting the **_DRY (Don't Repeat Yourself)_** programming principle. 👍